libs/libavro/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```

Comment:
Since you're using LEDE
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code
https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain

Might be of consideration